packages/video_player/tizen/src/video_player.h

```diff
@@ -48,7 +50,11 @@ class VideoPlayer {
   std::unique_ptr<flutter::EventChannel<flutter::EncodableValue>> eventChannel_;
   std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> eventSink_;
   long textureId_;
-  FlutterTextureRegistrar *textureRegistrar_;
+  flutter::TextureRegistrar *textureRegistrar_;
+  std::unique_ptr<flutter::TextureVariant> textureVariant_;
+  std::unique_ptr<FlutterDesktopGpuBuffer> flutterDesktopGpuBuffer_;
+  std::mutex mutex_;
+  media_packet_h mediaPacket_ = nullptr;
```

Comment:
```suggestion
  flutter::TextureRegistrar *texture_registrar_;
  std::unique_ptr<flutter::TextureVariant> texture_variant_;
  std::unique_ptr<FlutterDesktopGpuBuffer> flutter_desktop_gpu_buffer_;
  std::mutex mutex_;
  media_packet_h media_packet_ = nullptr;
```

Reply:
Thanks for your code review, I have update variable name.